config/install/core.entity_form_display.media.images_media_bundle.default.yml

```diff
@@ -0,0 +1,42 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.media.images_media_bundle.field_image
+    - image.style.thumbnail
+    - media_entity.bundle.images_media_bundle
```

Comment:
Let's name this bundle just "image" instead. Try to keep things as simple as possible.